101-vm-with-data-management-gateway/azuredeploy.json

```diff
@@ -0,0 +1,297 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "existingDataFactoryName": {
+      "type":"string",
+      "metadata": {
+        "description": "your existing data factory name"
+      }
+    },
+    "GatewayName": {
```

Comment:
Gateway name will be used in the pipeline if customers want to do a copy activity. So we need our customer to specify the gateway name.